src/shared/assets.ts

```diff
@@ -344,6 +344,18 @@ export class AssetGenerator {
 
         return result;
     }
+
+    public getAssetsPathAndProgram(): [string, string] {
```

Comment:
>() [](http://example.com/codeflow?start=34&length=2)

Shouldn't this require the path to the project instead of looping over all project?

Reply:
We need to loop all the projects, usually we are launching the server project but we need the assetsPath of the client project which is the webassembly one.

Reply:
Can we do something to avoid this? At least having an option, since a given workspace could have multiple projects and I am assuming we don't want to pick the wrong one.

Reply:
Done, we will continue passing the assets path but if for some reason the MVID of a loaded module is different from the one loaded from assets path we will load the metadata information from debugger protocol and ignore assets path.